### DISCUSSION TITLE
Can the asynchronous script be loaded without deleting the script tag?

### DISCUSSION THREAD

**Discussion by masx200** [webpack/webpack#13692] [General]
https://github.com/webpack/webpack/blob/c181294865dca01b28e6e316636fef5f2aad4eb6/lib/runtime/LoadScriptRuntimeModule.js#L140

Can the asynchronous script be loaded without deleting the script tag?

**Comment by alexander-akait** [comment]
What is use case?

**Reply by masx200** [reply]
> What is use case?



Then why delete the script tag?

There is a scenario where the script tag needs to be retained. When the user saves the complete web page, it will be opened locally afterwards.

**Reply by alexander-akait** [reply]
It is to avoid memory leading
